### DISCUSSION TITLE
Cloud Mask for Landsat

### DISCUSSION THREAD

**Discussion by sunwooy** [kvos/CoastSat#494] [Q&A] [answered]
Hi kvos,

Thanks for this amazing tool. Looks great and excited to explore it.

Saw a recent update note to improve cloud masking for Landsat. What kind of improvements were done? Thank you!

**Comment by sunwooy** [comment]
@2320sharon @DanieTheron

**Comment by DanieTheron** [comment, marked as answer]
Hi @sunwooy,
The cloud_mask function was improved/enhanced - refer to issue #407.
Hope this helps.
Regards.
